### PR TITLE
Update doc strings in matrices.py

### LINKED ISSUE #13449: Improve docstring of matrices/matrices.py
The docstring of functions in matrices/matrices.py looks ugly. Since it should use double backticks for functions mention instead of single backtick.

Look at PR #12726 .

### PR DESCRIPTION
**Pull request by vishalg2235**
Fixes #13449 
Ping @gxyd

### CODE REVIEW

**Comment by vishalg2235** [comment]
@gxyd had I done something wrong?

**Comment by gxyd** [comment]
Actually, initially there appeared one more pull request (#13461) fixing this same issue (I know you sent PR first, but I couldn't review).

Now there appears we needed some decision on the behavior of single and double backticks here in issue #13519. Read this Aaron's comment https://github.com/sympy/sympy/pull/13461#issuecomment-338560694 , you will need to follow the comments on PR https://github.com/sympy/sympy/issues/13519.

You can try to fix issue #13519 .

**Comment by vishalg2235** [comment]
Ok I got it and I will try to fix #13519

**Comment by vishalg2235** [comment]
@gxyd as per last comment in #13519 .Should I change `this` to render code and $this$ to render math part ?

File: sympy/matrices/matrices.py [hunk 1]
```diff
@@ -2628,15 +2628,15 @@ def gauss_jordan_solve(self, b, freevar=False):
...
+        Method to find the matrix inverse of ``K`` (mod ``m``) implemented in this function:
 
-        * Compute `\mathrm{adj}(K) = \mathrm{cof}(K)^t`, the adjoint matrix of `K`.
+        * Compute ``\mathrm{adj}(K) = \mathrm{cof}(K)^t``, the adjoint matrix of ``K``.
 
-        * Compute `r = 1/\mathrm{det}(K) \pmod m`.
+        * Compute ``r = 1/\mathrm{det}(K) \pmod m``.
 
-        * `K^{-1} = r\cdot \mathrm{adj}(K) \pmod m`.
+        * ``K^{-1} = r\cdot \mathrm{adj}(K) \pmod m``.
```
Comment (asmeurer) [inline sympy/matrices/matrices.py]: This should be math.

**Comment by asmeurer** [comment]
There are several instances of LaTeX math that should stay a single backquote here.

File: sympy/matrices/matrices.py (hunk 1 above)
Comment (vishalg2235) [reply sympy/matrices/matrices.py]: yup..

**Pull request closed without merging**